ui/src/message_popup/playtest/danson_experience_page.jsx

```diff
@@ -69,6 +69,32 @@ export default React.createClass({
     });
   },
 
+  onGetNextQuestion(questions:[QuestionT], responses:[ResponseT]) {
+    const allQuestions = questions.slice();
```

Comment:
👍  Nice, this is great so you can work with `questions` but not modify the value passed into this function.